web/ui/mantine-ui/src/components/SettingsMenu.tsx

```diff
@@ -103,22 +105,48 @@ const SettingsMenu: FC = () => {
                 />
               </Stack>
             </Fieldset>
-          </Stack>
-
-          <Stack>
-            <Fieldset p="md" legend="Alerts page settings">
+            <Fieldset p="md" legend="Targets page settings">
               <Checkbox
-                checked={showAnnotations}
-                label="Show expanded annotations"
+                checked={showEmptyPools}
+                label="Show empty pools"
                 onChange={(event) =>
                   dispatch(
                     updateSettings({
-                      showAnnotations: event.currentTarget.checked,
+                      showEmptyPools: event.currentTarget.checked,
                     })
                   )
                 }
               />
             </Fieldset>
+          </Stack>
+
+          <Stack>
+            <Fieldset p="md" legend="Alerts page settings">
```

Comment:
I also noticed that I introduced two "Alerts page settings" sections into the settings menu here earlier, which should be merged into one. I'll create a separate PR for that.